Tests/BuildTests/BuildPlanTests.swift

```diff
@@ -2551,6 +2551,10 @@ final class BuildPlanTests: XCTestCase {
                         TargetDescription(name: "StaticLibrary", path: "StaticLibrary.xcframework", type: .binary),
                     ]
                 ),
+            ],
+            binaryArtifacts: [
+                .init(kind: .xcframework, originURL: nil, path: AbsolutePath("/Pkg/Framework.xcframework")),
+                .init(kind: .xcframework, originURL: nil, path: AbsolutePath("/Pkg/StaticLibrary.xcframework"))
```

Comment:
this is now needed since we no longer that if the artifact is missing its local, which I think is better. @neonichu wdyt?